config/database.yml

```diff
@@ -1,130 +1,26 @@
-# PostgreSQL. Versions 9.3 and up are supported.
-#
-# Install the pg driver:
-#   gem install pg
-# On macOS with Homebrew:
-#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
-# On Windows:
-#   gem install pg
-#       Choose the win32 build.
-#       Install PostgreSQL and put its /bin directory on your path.
-#
-# Configure Using Gemfile
-# gem "pg"
-#
 default: &default
   adapter: postgresql
   encoding: unicode
-  host: <%= ENV.fetch('DATABASE_HOST') { 'localhost' } %>
```

Comment:
Looks like we are copying the same values between envs.
Why removing these default then?

Reply:
Heroku only needs the DATABASE_URL var for production. It doesn't need host/password/etc.

Reply:
I'll test that out and also try the changes Olivia suggested.